python/paddle/v2/fluid/tests/test_iou_similarity_op.py

```diff
@@ -0,0 +1,36 @@
+import unittest
+import numpy as np
+import sys
+import math
+from op_test import OpTest
+
+
+class TestIOUSimilarityOp(OpTest):
+    def set_data(self):
+        self.init_test_data()
+        self.inputs = {'X': self.boxes1, 'Y': self.boxes2}
+
+        self.outputs = {'Out': self.output}
+
+    def test_check_output(self):
+        self.check_output()
+
+    def test_check_grad(self):
+        return
+
+    def setUp(self):
+        self.op_type = "iou_similarity"
+        self.set_data()
```

Comment:
If only one test, the code in `set_data()` and `init_test_data()` can be moved here.